rfd/0007-rbac-oss.md

```diff
@@ -90,7 +90,11 @@ role:
    name: user
 spec:
   options:
-    port_forwarding: true
+    ssh_port_forwarding:
+      remote:
+        enabled: true
+      local:
+        enabled: true
```

Comment:
These RFD edits probably weren't necessary, but I figured it doesn't hurt to keep example configurations accurate